### ISSUE TITLE
Add accessibility language requirements to logo and slogan

### ISSUE THREAD

**Issue by HeleenSG** [minvws/nl-rdo-manon#255]
If a slogan or text within a logo is in a different language then the document it should be marked as such. Making it possible for accessibility tools to use the correct language when reading it out loud. 

E.g <code>lang="en"</code>
